report-api-faq.md

```diff
@@ -163,7 +163,10 @@ The payout details can be found with these endpoints, where `{topic}` is `funds`
 * [`GET:/report/v2/ledgers/{ledgerId}/{topic}/dates/{ledgerDate}`][fetch-report-by-date-endpoint]
 * [`GET:/report/v2/ledgers/{ledgerId}/{topic}/feed`][fetch-report-by-feed-endpoint]
 
-Match the settlement ID in the bank with `pspReference`.
+Match the `reference` in the bank statement with the `pspReference` in the Report API.
+When you retrieve a series of payments with
+[`GET:/report/v2/ledgers/{ledgerId}/{topic}/feed`](https://developer.vippsmobilepay.com/api/report/#tag/reportv2ledgers/paths/~1report~1v2~1ledgers~1%7BledgerId%7D~1%7Btopic%7D~1feed/get)
+in the Report API, each payment’s `pspReference` is the `reference` for the bank payout.
```

Comment:
No I don't understand this one...   what is a "payment" here?

Reply:
@dss-vipps Hi! Does he mean "Payment entries" or "payment reports" ? something like that? If you think of a better way, just go ahead and commit it and them merge this thing so it doesn't get too old while he's away. I'm going on vacation now, too. TaTa! :-)

Reply:
Tried to make it clearer. Did I succeed..?